### PR TITLE
New story added

### PR DESCRIPTION
**Pull request by usurper12**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1738)

### CODE REVIEW

File: english/batman/batman.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+wanna know a secret bruce wayne is the BAT
```
Comment (casey-collab) [inline english/batman/batman.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by usurper12** [comment]
done @casey-collab

**Comment by casey-collab** [comment]
Looks great! :smiley_cat:

**Pull request merged**